packages/components/src/InlineMessage/InlineMessage.scss

```diff
@@ -27,6 +26,12 @@ $tc-inline-message-icon-offset: 0.4rem !default;
 		line-height: $tc-inline-message-min-height;
 		min-height: $tc-inline-message-min-height;
 
+		span {
```

Comment:
could you add a css class on all instead, please?
I'm not sure than a span will stay to display link in the future, so maybe a class is better than a specific element based selector!

Reply:
ok you are not targeting the link, I've read too fast, but yes a css class will be better imho